### ISSUE TITLE
File Dialog messages that contain \r\n render the \r\n as text

### ISSUE THREAD

**Issue by mjjames** [dotnet/wpf#977]
* .NET Core Version: 3.0.0-preview6-27804-01
* Windows version: Windows 10 Build 18917.rs_prerelease.190607-1942 and Windows 7 sp1
* Does the bug reproduce also in WPF for .NET Framework 4.8?: No

 **Problem description:**
When you use an Open File Dialog and provide an invalid file OR if you use a Save File Dialog and try to overwrite an existing file the messagebox renders the \r\n as text instead of  a line break.

This happens if you use the Microsoft.Win32 base implementations or the System.Windows.Forms varients

 **Actual behavior:** 
![image](https://user-images.githubusercontent.com/514455/59556046-22aa9280-8fb4-11e9-8656-53ceb7edb051.png)
 
 **Expected behavior:**
![image](https://user-images.githubusercontent.com/514455/59556078-9f3d7100-8fb4-11e9-83bc-27e8d45027db.png)

 **Minimal repro:**
```
var saveDialog = new Microsoft.Win32.SaveFileDialog
{
	OverwritePrompt = true
};
var success = saveDialog.ShowDialog();
var saveDialogForms = new System.Windows.Forms.SaveFileDialog
{
	OverwritePrompt = true
};
var successForms = saveDialog.ShowDialog();
```

**Comment by mjjames** [comment]
Appears to just be an issue with the [strings resource file](https://github.com/dotnet/wpf/blob/ae1790531c3b993b56eba8b1f0dd395a3ed7de75/src/Microsoft.DotNet.Wpf/src/PresentationFramework/Resources/Strings.resx) All of the other resource strings just use \n however the file dialog ones still use the \r\n format

**Comment by weltkante** [comment]
This analysis/fix doesn't make sense. If `\n` would work then you'd see a line break after a literal `\r`

It might be worth digging deeper to find the actual bug (or add a link to the issue if you already know *why* this happens)

**Comment by rjvdboon** [comment]
I think the .resx files should not contain the literal `\r\n`, but have either xml encoded entities, or actual newline in the value. If so, the root cause is in the original creation of the .resx file.

Note that .Net framework sdk resgen.exe translates from .txt to .resx by replacing escape sequences with the actual value, i.e.
`WithNewLine=Before\r\nAfter`
is translated to
```
<data name="WithNewLine" xml:space="preserve">
    <value>Before
After</value>
  </data>
```

See also [Resource text files](https://docs.microsoft.com/en-us/dotnet/framework/resources/creating-resource-files-for-desktop-apps#resources-in-text-files)

**Comment by weltkante** [comment]
WinForms solved this in PR dotnet/winforms#1211 by replacing escape sequences with literal line breaks.

**Comment by RussKie** [comment]
You may wish to sweep all your resources for `\r\n` sequence and replace them with hard line breaks.
